lib/final_destination.rb

```diff
@@ -505,4 +540,8 @@ def fetch_canonical_url(body)
 
     uri(complete_url)
   end
+
+  def blocked_domain?(uri)
+    uri && Onebox::DomainChecker.is_blocked?(uri.hostname)
```

Comment:
Should we modify `Onebox::DomainChecker.is_blocked?` to check presence instead?

Reply:
Actually, I've reviewed the code and realized  it's not possible for `uri` here to be nil because before this method is called there's a validation that ensures the `uri` isn't nil. I've removed the `uri` check from the condition.